frontend/package.json

```diff
@@ -1,21 +1,45 @@
 {
-  "name": "ecqm-dedupe-desktop-app",
+  "name": "deduplifhir-app",
   "version": "1.0.0",
-  "description": "",
+  "productName": "DedupliFHIR",
+  "description": "An Electron app that uses the DedupliFHIR CLI to provide data-linkage and AI deduplication for reported ACO data at scale.",
+  "author": "Natalia Luzuriaga",
+  "license": "CC0-1.0",
   "main": "main.js",
   "scripts": {
     "start": "electron .",
     "lint": "eslint ./**.js",
     "lint:fix": "eslint --quiet --fix \"**.js\"",
     "prettier": "prettier --check ./**.{js,css}",
     "prettier:fix": "prettier --write \"./**.{js,css}\"",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "pack": "electron-builder --dir",
+    "dist": "../set-up-python-env.sh && electron-builder"
```

Comment:
It might be more straightforward to use the Makefile for this overall step if there are multiple pieces involved?